contracts/EthereumDIDRegistry.sol

```diff
@@ -4,6 +4,7 @@ contract EthereumDIDRegistry {
 
   mapping(address => address) public owners;
   mapping(address => mapping(bytes32 => mapping(address => uint))) public delegates;
+  mapping(address => mapping(bytes32 => mapping(address => uint))) public unrevokableUntil;
```

Comment:
If you can revoke it in the future it's not really unrevokable? How would a state channel know how far in the future it should enforce the unrevokability?